Library/Formula/stash-cli.rb

```diff
@@ -0,0 +1,17 @@
+class StashCli < Formula
+  homepage "https://marketplace.atlassian.com/plugins/org.swift.stash.cli"
+  url "https://bobswift.atlassian.net/wiki/download/attachments/16285777/stash-cli-3.9.0-distribution.zip?api=v2"
+  version "3.9.0"
+  sha1 "f615519894b194959754b9a7b5fb9bc03855dbcd"
+
+  def install
+    inreplace "stash.sh", "`dirname $0`", share
+    share.install "lib"
+    bin.install "./stash.sh"
+    File.rename prefix/"bin/stash.sh", prefix/"bin/stash"
+  end
+
+  test do
+    assert shell_output(bin/"stash --help 2>&1 | head").include?("Usage:")
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!